docs/stratis.txt

```diff
@@ -42,11 +42,20 @@ pool destroy <pool_name>::
 pool add-data <pool_name> <blockdev> [<blockdev>..]::
 	 Add one or more blockdevs to an existing pool, to enlarge its storage
 	 capacity.
+pool init-cache <pool_name> <blockdev> [<blockdev>..]::
+	 Initialize a cache for an existing pool.
+	 Add one or more blockdev to a pool, to be used as cache
+	 instead of additional storage. Typically, smaller and faster
+	 drives such as SSDs, are used for this purpose.
+	 NOTE: There is a temporary restriction on the total size of the
+         cache device of 32 TiB. Adding devices to the cache so that the
+         cumulative size of all the devices in the cache exceeds 32 TiB will
+         result in an error.
 pool add-cache <pool_name> <blockdev> [<blockdev>..]::
-	 Add one or more blockdevs to an existing pool, to be used as a cache
-	 instead of additional storage. Typically, smaller and faster drives,
-	 such as SSDs, are used for this purpose.
-         NOTE: There is a temporary restriction on the total size of the
+	 Add one or more blockdevs to an existing pool with an initialized cache.
+	 Typically, smaller and faster drives, such as SSDs,
```

Comment:
You should feel free to drop this line for add-cache, and consider it moved to ```init-cache```.